server/src/main/resources/db/common/V103__alter_member_profile_table.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE member_profile ADD COLUMN last_seen TIMESTAMP;
```

Comment:
Your revision should be V104, and remove_userid_from_settings_table.sql should remain as V103 or you will likely run into some migration issues when you merge into develop since the latter has already been merged there as V103_remove_userid_from_settings_table.sql.

Reply:
Thanks for catching that I got them switched. Fixed.